code/modules/mob/living/life.dm

```diff
@@ -291,18 +277,22 @@
 	if(drip_amount)
 		touching_reagents.trans_to(loc, drip_amount)
 
-/mob/living/proc/handle_weather_effects(obj/abstract/weather_system/weather)
-	// Handle physical effects of weather.
-	if(!istype(weather))
-		return
-	var/decl/state/weather/weather_state = weather.weather_system.current_state
-	if(istype(weather_state))
-		weather_state.handle_exposure(src, get_weather_exposure(weather), weather)
+/mob/living/process_weather(obj/abstract/weather_system/weather, decl/state/weather/weather_state)
+	// Handle physical effects of weather. Ambience is handled in handle_environment
+	// as clients should get wind noise etc. regardless of weather sensitivity.
```

Comment:
i don't think this is correct, i think `WEATHER_IGNORE` should block ambience, while `WEATHER_EXPOSED`, `WEATHER_ROOFED`, and `WEATHER_PROTECTION` should allow it

Reply:
I'm pretty sure `WEATHER_IGNORE` is specifically for "you are two z-levels down and there is no way the weather could reach you," IIRC? Like how we wouldn't want rain sounds to be audible from within the deep caverns

Reply:
was just replying to `Dunno if I like that on reflection, it'd be nice to hear rain from inside a locker or tent or such.`, no request for changes. as evidenced by how there were not changes requested